src/angular/styles/_typography.scss

```diff
@@ -2437,12 +2437,13 @@ $menuItemPaddingTopBottom: 4;
     @include user-select(none);
   }
 
-  &:is(:not(:disabled):not([disabled]):not(.sbb-disabled):is(:hover, :focus, :hover strong, :focus
-        strong), .sbb-focused) {
-    color: var(--sbb-color-red);
+  &:is(:not(:disabled):not([disabled]):not(.sbb-disabled):is(:hover, :focus), .sbb-focused) {
```

Comment:
The generated output is a little bit long, but I don't see a good alternative...

generated output:
`:is(.sbb-menu-item:is(:not(:disabled):not([disabled]):not(.sbb-disabled):is(:hover, :focus), .sbb-focused),
      .sbb-menu-item:is(:not(:disabled):not([disabled]):not(.sbb-disabled):is(:hover, :focus), .sbb-focused) strong)`

Reply:
I agree, but in this case I think we just need to live with it.